src/durabletask/azext_durabletask/tests/latest/test_durabletask.py

```diff
@@ -0,0 +1,13 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.testsdk import *
+
+
+class DurabletaskScenario(ScenarioTest):
+    def test_help(self):
+        pass
```

Comment:
Please add tests for the commands

Reply:
Since our service is still in private preview and being finalized/stabilized, I don't believe we should have testing at this time due to the possibility of a failure occurring on the underlying service. Furthermore, subscriptions must be first registered and then approved for this service.